docs/SECURITY.md

```diff
@@ -0,0 +1,81 @@
+# EVE Security Overview
+
+## Introduction
+
+What makes EVE a secure-by-design system is that it has been developed from the ground up with security in mind. EVE doesn't rely on 3d party or add-on components to provide trust in the system, but rather offers a set of well-define principles on which a trust model can be built. You will notice that EVE doesn't call itself a trusted system -- but rather a trustworthy one. The distinction is subtle, but very important and was first articulated by [Joanna Rutkowska](https://www.darkreading.com/vulnerabilities---threats/rutkowska-trust-makes-us-vulnerable/d/d-id/1330587). Joanna, of course, is the architect of [Qubes OS](https://www.qubes-os.org) with wich EVE shares a good deal of core security principles like [security by compartmentalization](https://www.qubes-os.org/doc/glossary/#qubes-os) and focusing on representing all user applications as VM-based abstractions (Qubes OS calls them [Qubes](https://www.qubes-os.org/doc/glossary/#qube), EVE calls them Edge Containers](ECOS.md)).
+
+We have made an effort to provide users of EVE with a system that is both practically secure and can be deployed in a zero-touch fashion. In order to achieve our goals we put forward a few guiding principles (some of them borrowed from Chromium's approach to securing mobile devices):
+
+* Make Edge Nodes secure-by-default
+* Defense in depth (based on the hypervisor boundary protection guarantees)
+* Relying on state-of-the-art hardware elements (such as TPM and TEE) for providing robust root-of-trust
+* Remote attestation and measured boot
+* Robust trust model between EVE and its controller
+* X509-based identity assigned to all the actors in the system
+* Workloads which are immutable
+* Secure overlay network
+
+In the rest of this document, we explain these principles and discuss some expected use cases for Edge Node devices running EVE.  We then give a high-level overview of the threat model against which we will endeavor to protect our users, while still enabling them to make full use of their cloud-based controller orchestration service.
+
+## EVE deployment model
+
+Reasoning about security and trustworthiness of any system is only meaningful if one understands how the systems is going to be deployed and used. EVE main deployment target are Edge Nodes. Typically, Edge Nodes are small form factor, ruggedized PC-like systems that are deployed in physically insecure environments with ad-hoc networking without any firewall-like guarantees. This type of environment is very similar to the one assumed by mobile computing systems (laptops, tablets and smart phones) and it is not a coincidence that EVE shares a lot of security principles with operating systems developed for mobile computing (e.g. Chromium OS and Android). For example, EVE's approach to thwarting physical attacks and protecting the integrity of the software stack running on the Edge Node 
```

Comment:
Typo: "system is going to be deployed"

Reply:
Typo: "EVE main deployment targets"

Reply:
Question: "PC-like" implies that it is like a desktop; aren't these more like a mini server, or at least in people's minds?

Reply:
Question: last sentence in paragraph looks like it is missing an ending?

Reply:
Well, they are called Industrial PCs in a lot of cases.

Reply:
Re: missing ending -- that actually got fixed in my force push -- but you may have reviewed before that.